app/javascript/components/request-workflow-status/data.js

```diff
@@ -59,6 +59,15 @@ export const workflowStatusData = (response) => {
     return undefined;
   }
   const rows = response.context ? rowData(response.context) : [];
+  if (response.context && response.context.State) {
+    rows.push({
+      id: `${response.context.State.Guid.toString()}+${Math.random()}`,
+      name: response.context.State.Name,
+      enteredTime: convertDate(response.context.State.EnteredTime.toString()),
+      finishedTime: convertDate(response.context.State.FinishedTime.toString()),
```

Comment:
If it is currently running it shouldn't have a FinishedTime

Reply:
nor a duration

Reply:
actually maybe it should have a duration on presentation?  not sure.

Reply:
Yeah I was thinking duration could be Time.now - StartTime and indicate how long it has been running for.